<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PenguinView</class>
 <widget class="QMainWindow" name="PenguinView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PenguinView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>320</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonAbout">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/aboutLight.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonCast">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/penguinLight.gif</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonLoad">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/raspberryPiLight.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidgetVeiwWindow">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Audiowide</family>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageLoad">
       <widget class="QLabel" name="labelLoad">
        <property name="geometry">
         <rect>
          <x>128</x>
          <y>6</y>
          <width>168</width>
          <height>27</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>4</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Audiowide</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Load / Select </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelSelectDevice">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>90</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Audiowide</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Choose</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidgetDeviceList">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>48</y>
          <width>341</width>
          <height>111</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>4</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Audiowide</family>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <item>
         <property name="text">
          <string>Device List:</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dev1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dev2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dev3</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="labelOpenFIle">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Audiowide</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Open File</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditOpenFile">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>180</y>
          <width>341</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonBrowse">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>210</y>
          <width>75</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Audiowide</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="playLocalButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>240</y>
          <width>110</width>
          <height>40</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Audiowide</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Play Local</string>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="labelStream">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>285</y>
          <width>254</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Audiowide</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Or stream your favorite video!</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelURL">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Audiowide</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>URL</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditStream">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>320</y>
          <width>341</width>
          <height>25</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>4</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Audiowide</family>
          <pointsize>11</pointsize>
          <italic>false</italic>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Press Enter to Stream</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelSelectDevice_2">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>110</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Audiowide</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Device:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <zorder>labelLoad</zorder>
       <zorder>labelSelectDevice</zorder>
       <zorder>listWidgetDeviceList</zorder>
       <zorder>labelOpenFIle</zorder>
       <zorder>lineEditOpenFile</zorder>
       <zorder>pushButtonBrowse</zorder>
       <zorder>playLocalButton</zorder>
       <zorder>labelStream</zorder>
       <zorder>labelURL</zorder>
       <zorder>lineEditStream</zorder>
       <zorder>labelSelectDevice_2</zorder>
       <zorder>frame</zorder>
      </widget>
      <widget class="QWidget" name="pageContact">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="labelContactUS">
          <property name="font">
           <font>
            <family>Audiowide</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Contact Us</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonSendEmail">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Audiowide</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditEnterEmail">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Audiowide</family>
            <pointsize>12</pointsize>
            <italic>false</italic>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Enter your email here</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Audiowide</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>The team at Air_Penguin thanks you for your feedback.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageAbout">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="labelAbout">
          <property name="font">
           <font>
            <family>Audiowide</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>About</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollAreaAbout">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>88</width>
             <height>88</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QPlainTextEdit" name="plainTextEditAbout">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="plainText">
               <string>##########################################
#############____Air_Penguin____############
##########################################

Designed by: 
	-&gt; John Nistico
	-&gt; Tony Bright
	-&gt; Brad Stemm
	-&gt; Melissa Lynch
	-&gt; Emile Tawfik
	-&gt; Denis Le Borgne
	-&gt; Jordan Humphrey

Server Implementation:
	--&gt; Emile Tawfik
	--&gt; Denis Le Borgne
	they are wizards of c++ and git

Backend and Implementation:
	--&gt; John Nistico
	* Endless Debugging
	* Keeping the backend together

User Interface:
	--&gt;David Bright
	* widgets and buttons
	* keeping it snappy
Presentation:
	--&gt; Melissa Lynch
	* Eye for minimalism
	* Pooling resources for presentation

	--&gt; Brad Stemm
	* for the incredible logo
	* getting the Peng to fly on the pi

Testing:
	--&gt; Jordan Humphrey
	*keeping the Peng watertight
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Audiowide</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuAirPenguin">
    <property name="font">
     <font>
      <family>Audiowide</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>AirPenguin</string>
    </property>
    <widget class="QMenu" name="menuThemes">
     <property name="font">
      <font>
       <family>Audiowide</family>
      </font>
     </property>
     <property name="title">
      <string>Themes</string>
     </property>
     <addaction name="actionLight_Theme"/>
     <addaction name="actionDark_Theme"/>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuAutoPlay">
     <property name="font">
      <font>
       <family>Audiowide</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>AutoPlay</string>
     </property>
     <addaction name="actionOn"/>
    </widget>
    <widget class="QMenu" name="menuDrag_Drop">
     <property name="title">
      <string>Drag &amp; Drop</string>
     </property>
     <widget class="QMenu" name="menuAutoLaunch">
      <property name="title">
       <string>AutoLaunch</string>
      </property>
      <addaction name="AutoLaunchOn"/>
     </widget>
     <addaction name="menuAutoLaunch"/>
    </widget>
    <addaction name="menuAutoPlay"/>
    <addaction name="menuThemes"/>
    <addaction name="actionContact"/>
    <addaction name="menuDrag_Drop"/>
   </widget>
   <addaction name="menuAirPenguin"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="toolTip">
    <string>Options</string>
   </property>
  </action>
  <action name="actionLight_Theme">
   <property name="text">
    <string>Light Theme</string>
   </property>
   <property name="font">
    <font>
     <family>Audiowide</family>
    </font>
   </property>
  </action>
  <action name="actionDark_Theme">
   <property name="text">
    <string>Dark Theme</string>
   </property>
   <property name="font">
    <font>
     <family>Audiowide</family>
    </font>
   </property>
  </action>
  <action name="actionContact">
   <property name="text">
    <string>Contact Us</string>
   </property>
   <property name="font">
    <font>
     <family>Audiowide</family>
    </font>
   </property>
  </action>
  <action name="actionOn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="font">
    <font>
     <family>Audiowide</family>
    </font>
   </property>
  </action>
  <action name="actionOff">
   <property name="text">
    <string>Off</string>
   </property>
   <property name="font">
    <font>
     <family>Audiowide</family>
    </font>
   </property>
  </action>
  <action name="AutoLaunchOn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>On</string>
   </property>
  </action>
  <action name="AutoLaunchOff">
   <property name="text">
    <string>Off</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>AutoLaunchOn</sender>
   <signal>triggered()</signal>
   <receiver>PenguinView</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AutoLaunchOff</sender>
   <signal>triggered()</signal>
   <receiver>PenguinView</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
